### PR TITLE
Add additional unlock_time details to Moneropedia

### PR DESCRIPTION
**Pull request by SamsungGalaxyPlayer**
Requires a review by a knowledgeable individual, such as @TheCharlatan or @Mitchellpkt.

### CODE REVIEW

**Review by TheCharlatan — COMMENTED** [review]
Thanks for adding this. I just have a suggestion for additional content.

File: _i18n/en/resources/moneropedia/unlocktime.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
 ---
```
Comment (ChanarinThintip) [inline _i18n/en/resources/moneropedia/unlocktime.md:1]: _i18n/en/resources/moneropedia/unlocktime.md #1343

**Comment by Mitchellpkt** [comment]
Good updates, thanks for writing this up sgp.

Hmm, I'm not sure if this is in scope of the doc, but I'll mention it just in case - there is a mismatch between CLI & RPC behavior around transaction generation. If I recall correctly, the wallet only accepts height-like values (up to millions), but the RPC accepts any uint64 value (up to 18,446,744,073,709,551,615). Thus transactions with very high lock values were either submitted by RPC or custom software.

**Comment by Mitchellpkt** [comment]
Oh, and also potential wording tweak, clarify
"_The integer should be an ***absolute*** block height value or Unix epoch timestamp, not a relative value_"
to
"_The integer value will be interpreted by the protocol as an ***absolute*** block height value or Unix epoch timestamp, not a relative value_"

**Comment by SamsungGalaxyPlayer** [comment]
@Mitchellpkt @TheCharlatan please review when you have a chance

**Review by TheCharlatan — APPROVED** [review]

**Comment by erciccione** [comment]
Let's wait a bit before merging this. #1270 must be kept in consideration.

**Comment by plowsof** [comment]
closed by #2076

**Pull request closed without merging**